pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/kit/KitTrackingScanUseCase.java

```diff
@@ -17,22 +19,8 @@ public KitTrackingScanUseCase(KitPayload kitPayload, KitDao kitDao) {
 
     @Override
     protected Optional<ScanError> process(ScanPayload scanPayload) {
-        String kitLabel = scanPayload.getKitLabel();
```

Comment:
This logic has moved into the dao, so that we can test more code without having to hit the route.